exec/utils_test.go

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024-Present Defense Unicorns
+
+package exec
+
+import (
+	"testing"
+)
+
+func TestMutateCommand(t *testing.T) {
+	type test struct {
+		cmd   string
+		shell ShellPreference
```

Comment:
rename symbols for `ShellPreference` type to be `shellPref` or `pref` to increase clarity